website/content/docs/secrets/kmip-profiles.mdx

```diff
@@ -171,8 +171,72 @@ Vault implements version 1.4 of the following Key Management Interoperability Pr
+    | Process Start Date [KMIP-SPEC 3.25][kmip-spec-3.25]                    | ✅        | Vault 1.11 |
+    | Protect Stop Date [KMIP-SPEC 3.26][kmip-spec-3.26]                     | ✅        | Vault 1.11 |
+
+  5. Supports the following message encoding:
+
+    | Message Encoding                                                                     | Supported | Notes |
+    | -------------------------------------------------------------------------------------| :--------:|:-----:|
+    | Cryptographic Algorithm [KMIP-SPEC 9.1.3.2.13][kmip-spec-9.1.3.2.13] with values:    |           |       |
+    | i. RSA                                                                               | ✅        | Vault 1.13 |
+    | Object Type [KMIP-SPEC 9.1.3.2.12][kmip-spec-9.1.3.2.12] with value:                 |           |        |
+    | i. Public	 Key                                                                       | ✅        | Vault 1.13 |
+    | ii. Private	 Key                                                                     | ✅        | Vault 1.13 |
+    | Key Format Type [KMIP-SPEC 9.1.3.2.3][kmip-spec-9.1.3.2.3] with value:               |           |        |
+    | i. PKCS#1                                                                            | ✅        | Vault 1.13 <br/> Supported for Private and Public Keys|
+    | ii. PKCS#8                                                                           | ✅        | Vault 1.13 <br/> Supported for Private Key|
+    | iii. Transparent RSA Public Key                                                      | ✅        | Vault 1.13 |
+    | iv. Transparent RSA Private Key                                                      | ✅        | Vault 1.13 |
+
+  6. MAY support any clause within [KMIP-SPEC][kmip-spec] provided it does not conflict with any other clause within the section [Symmetric Key Lifecycle Server][lifecycle-server]
+  7. MAY support extensions outside the scope of this standard (e.g., vendor extensions, conformance clauses) that do not contradict any KMIP requirements.
+
+## [Advanced Cryptographic Server][advanced-cryptographic-server]
+  1. SHALL conform to the [Baseline Server][baseline-server]
+  2. Supports the following client-to-server operations:
+
+    | Operation                                             | Supported | Notes   |
+    | ------------------------------------------------------| :--------:| --------|
+    | Encrypt [KMIP-SPEC 4.29][kmip-spec-4.29]              | ✅        | Vault 1.11 <br/> Supported for AES, unsupported for 3DES: <br/><br/> Supported Block Cipher Modes: <br/> <ol> <li> GCM </li> <li> CBC </li> <li> CFB </li> <li> CTR </li> <li> ECB </li> <li> OFB </li> </ol> <br/> Stream operations are supported except for GCM block cipher mode. <br/><br/> Supported padding methods: <br/> <ol> <li> None </li> <li> PKCS5 </li> </ol> <br/> Vault 1.13 <br/> Supported for RSA Asymmetric Keys  |
```

Comment:
Since the notes for Encrypt/Decrypt are the same as in the Basic Cryptographic Server section, what about changing the notes on that profile to "see Advanced Cryptographic Server" (and make that a link)?